.pre-commit-config.yaml

```diff
@@ -6,7 +6,9 @@ exclude: >
       \.vscode/settings\.json|
       tests/commonmark/commonmark\.json|
       .*\.xml|
-      tests/.*\.txt
+      tests/.*\.txt|
+      tests/test_parser.py|
+      tests/test_text_based.py
```

Comment:
Lazy 😜 ! Add `# noqa: E501` to specific lines that are too long, not ignore the whole file

Reply:
ahhh that's right, I knew there was a way to do that